src/openforms/emails/tests/test_digest_functions.py

```diff
@@ -392,3 +398,83 @@ def test_invalid_bag_configuration_without_required_property_is_not_collected(
         broken_configuration = collect_broken_configurations()
 
         self.assertEqual(broken_configuration, [])
+
+
+@override_settings(LANGUAGE_CODE="en")
+class InvalidCertificatesTests(TestCase):
+
+    def test_expiring_certificates_not_used_by_a_service_are_not_collected(self):
+        with open(TEST_FILES / "test.certificate", "r") as client_certificate_f:
+            CertificateFactory.create(
+                label="Test certificate",
+                public_certificate=File(client_certificate_f, name="test.certificate"),
+            )
```

Comment:
What would make this certificate turn up as invalid? Can you add a comment if it has a particular expiry date? Right now I'd say "yeah but even if this certificate is used by a service, it probably won't show because it's valid anyway".

Reply:
Yes, I had a comment in each test that was using the certificate concerning the expiry date (which would make it invalid). Is it ok if I add a comment at the class level, or should I add it to each test separately?

Reply:
I prefer to have all information in each test, that's consistent with the principle that each test should be debuggable/isolated by itself. Ideally to understand what's being tested, I only have to look at the test method itself.